web/src/main/java/it/pagopa/selfcare/mscore/web/model/delegation/DelegationRequest.java

```diff
@@ -0,0 +1,22 @@
+package it.pagopa.selfcare.mscore.web.model.delegation;
+
+import it.pagopa.selfcare.mscore.constant.DelegationType;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+public class DelegationRequest {
+
+    @NotNull
```

Comment:
Would be better to use NotBlank instead of NotNull, NotNull allows blank strings